### ISSUE TITLE
Insufficient color contrast ratio in MCQ option

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/mobile#1468]
## Description

When selected, the MCQ option is highlight with a white background. This causes an a11y issue if the option text is wrapped by `<code>` as the `<code>` content is also highlighted with a bright background color.

<img width="403" alt="Image" src="https://github.com/user-attachments/assets/66ccbc83-b841-4181-8bbd-03ed9ce72b68" />

I think we probably could just remove the white background and rely on the dot on the radio button to indicate the selected state, just like the web version:

<img width="778" alt="Image" src="https://github.com/user-attachments/assets/e853f439-aa50-4823-9dc7-93b00fbcf2e0" />
